catalog.yml

```diff
@@ -183,3 +183,7 @@
 - repo_url: https://github.com/greenelab/mpmp-manuscript
   html_url: https://greenelab.github.io/mpmp-manuscript/
   preprint_citation: doi:10.1101/2021.10.27.466140
+- repo_url: https://github.com/aseedb/synaptic_tomo_ms
+  html_url: https://aseedb.github.io/synaptic_tomo_ms/
+  preprint_citation: doi:10.1101/2022.03.07.483217
+  thumbnail_url: https://raw.githubusercontent.com/aseedb/synaptic_tomo_ms/main/thumbnail.png
```

Comment:
```suggestion
```
Because this thumbnail is named `thumbnail.png` and in the the manuscript repository, it will be detected automatically.

Reply:
somehow the thumbnail is not showing in the catalog.

Reply:
Have you completed a successful manuscript build after uploading the thumbnail image?  It looks like the image was added 4 days ago but both builds since then were cancelled.  Manubot needs to complete a build in order to detect the thumbnail image and set the `thumbnail_url` variable in https://github.com/aseedb/synaptic_tomo_ms/blob/output/variables.json

Reply:
You spotted the problem! I did not build after uploading the thumbnail image. Yesterday I have rebuilt and `thumbnail_url` is now correctly set in variables.json. The thumbnail does not yet appear in the catalog. Does it need to be rebuilt as well?

Reply:
Yeah, the catalog will rebuild automatically in less than one week